packages/block-editor/src/components/url-popover/style.scss

```
@@ -50,10 +62,31 @@
 }
 
 .block-editor-url-popover__settings {
+	display: block;
 	padding: $panel-padding;
 	border-top: $border-width solid $light-gray-500;
 
 	.components-base-control:last-child .components-base-control__field {
```

```suggestion
	.components-base-control:last-child,
	.components-base-control:last-child .components-base-control__field {
```

I think this should take care of @mapk's [comment below](https://github.com/WordPress/gutenberg/pull/15570#issuecomment-497140835).

Once this is fixed, I think this PR is good to go. 👍